photosynthesis_metrics/__init__.py

```diff
@@ -3,3 +3,4 @@
 from .fid import FID
 from .kid import KID
 from .tv import TVLoss, total_variation
+from .inception_score import IS
```

Comment:
Why the file with Inception Score is named `inception_score.py` but not `is.py`?

Reply:
Because then in `__init__.py` when I try to write
`from .is import IS, inception_score` Python throws an error for incorrect syntax.